components/eventing-controller/README.md

```diff
@@ -2,40 +2,49 @@
 
 ## Overview
 
-This component contains controllers for various CustomResourceDefinitions related to Eventing in Kyma. The following controller comes with this container:
+The Eventing Controller components contain reconcilers for various CRDs related to Eventing in Kyma:
 
-- [`controller`](https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/cmd/eventing-controller/main.go) which lays down the Eventing infrastructure in Business Event Bus (BEB) or [NATS](https://docs.nats.io/nats-concepts/intro).
+- Subscription reconciler, which lays down the Eventing infrastructure in Business Event Bus (BEB) or [NATS](https://docs.nats.io/nats-concepts/intro)
+- Backend reconciler, which updates the status of the currently active Eventing backend
+
+For eventing in NATS mode, Eventing Controller also acts as a dispatcher. It converts messages received from NATS to POST requests and forwards them to the configured sinks.
 
 ## Prerequisites
 
-- Install [ko](https://github.com/ko-build/ko/blob/main/docs/install.md) which is used to build and deploy the controller during local development
-- Install [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) which is the base framework for this controller (currently used version: 3.1)
-- Install [kustomize](https://github.com/kubernetes-sigs/kustomize) which lets you customize raw, template-free `yaml` files during local development
-- Install lint on the local environment
-```bash
-curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin
-```
+- Install [ko](https://github.com/ko-build/ko/blob/main/docs/install.md), which is used to build and deploy the controller during local development.
+- Install [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder), which is the base framework for this controller (currently used version: 3.1).
+- Install [kustomize](https://github.com/kubernetes-sigs/kustomize), which lets you customize raw, template-free `yaml` files during local development.
```

Comment:
Kubebuilder and Kustomize are capitalized later on in the document, so maybe they also should be here (?)

Reply:
true! done with https://github.com/kyma-project/kyma/pull/15907#:~:text=consistent%20capitalization-,7523183,-Merge%20state